src/handlebars/faq.handlebars

```diff
@@ -242,6 +242,65 @@
+    </div>
+
+    <div class="margin-bottom">
+    <p>Support for JavaFX/OpenJFX is a hot topic, and we receive many questions about it. This section tries to
+         answer the most important ones from the perspective of AdoptOpenJDK.</p>
+
+    <dl>
+    <dt> <strong>Does AdoptOpenJDK offer OpenJDK binaries with JavaFX support?</strong>
+    <dd> <p>No.</p>
+    <dt> <strong>Are there any plans to bundle OpenJFX with AdoptOpenJDK's binaries?</strong>
+    <dd> <p>No.</p>
+    <dt> <strong>Are there any plans to build standalone binaries of OpenJFX at AdoptOpenJDK?</strong>
+    <dd> <p>No.</p>
+    <dt> <strong>Why does AdoptOpenJDK not include OpenJFX?</strong>
+    <dd> <p><a href="https://blogs.oracle.com/java-platform-group/the-future-of-javafx-and-other-java-client-roadmap-updates">JavaFX
+         has been unbundled from OpenJDK</a> and now exists as a separate project called
+         <a href="https://wiki.openjdk.java.net/display/OpenJFX/Main">OpenJFX</a> with a different release schedule and
+         support policy. Contrary to OpenJDK, there is no open-source long term support for any OpenJFX version. While
+         <a href="https://wiki.openjdk.java.net/display/jdk8u">OpenJDK 8 receives updates for the years to
+         come</a>, OpenJFX 8 is unmaintained and contains known security issues. OpenJFX 11 and newer only get patches
+         for 6 months after their initial release. This situation makes it impossible for AdoptOpenJDK to provide
+         high-quality, up-to-date binaries of OpenJDK LTS releases (8 and 11 at the time of writing) with a bundled
+         OpenJFX.</p>
+
+         <p>As a result, we could only provide OpenJFX binaries for the most recent OpenJDK version. But because the
+         <a href="https://openjfx.io">OpenJFX project</a> already does that itself, we want to focus our resources on
+         OpenJDK.</p>
+    <dt> <strong>What are my options if I need to run a program that uses JavaFX?</strong>
+    <dd> <p>The <a href="https://openjfx.io/">OpenJFX project</a> provides binaries of OpenJFX that work
+         great together with AdoptOpenJDK. See the <a href="https://openjfx.io/openjfx-docs/#install-java">OpenJFX
+         documentation for instructions on how to get started</a>.</p>
+    <dt> <strong>What are my options if I need patches or support for JavaFX 8?</strong>
+    <dd> <p>Buy a license for Oracle JDK 8.
+         <a href="https://www.oracle.com/java/technologies/java-se-support-roadmap.html">Oracle provides support for
+         JavaFX 8 until March 2022</a>.</p>
+
+         <p>There are versions of <a href="https://www.azul.com/downloads/zulu-community/">Azul Zulu</a> and
+         <a href="https://bell-sw.com/java8">BellSoft Liberica</a> that include JavaFX and are available free of
+         charge. We cannot vouch for their quality. Reports by our community indicate that their offerings vary and
+         might not include all patches.</p>
```

Comment:
Remove final sentence.

Reply:
Same comment I made elsewhere :-) PR in https://github.com/AdoptOpenJDK/openjdk-website/pull/699/files